homeassistant/components/plant.py

```diff
@@ -179,10 +190,16 @@ def state_changed(self, entity_id, _, new_state):
         elif reading == READING_CONDUCTIVITY:
             self._conductivity = int(float(value))
         elif reading == READING_BRIGHTNESS:
+            self._brightness_history.add_measurement(self._brightness,
+                                                     new_state.last_updated)
             self._brightness = int(float(value))
         else:
             raise _LOGGER.error("Unknown reading from sensor %s: %s",
                                 entity_id, value)
+        if new_state.attributes.get(CONF_UNIT_OF_MEASUREMENT, None) \
```

Comment:
- Use `ATTR_UNIT_OF_MEASUREMENT`
- `None` is the default second argument when none is passed in. Since you're not using the return value, you can just test if the key exists with `if ATTR_UNIT_OF_MEASUREMENT in new_state.attributes:`

Reply:
ACK

Reply:
ACK